egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -257,6 +279,55 @@ def AddLpFilter(config_lines, name, input, rate, num_lpfilter_taps, lpfilt_filen
 
     return [tdnn_input_descriptor, filter_context, filter_context]
 
+def AddConvMaxpLayer(config_lines, name, input, args):
+    input = nodes.AddConvolutionLayer(config_lines, name, input,
```

Comment:
If input dictionary does not have the required keys raise an exception that the input needs the 3d-dim parameters. A key error which would otherwise be generated would be confusing.